### PR TITLE
Support Search forms

### PR DESCRIPTION
**Pull request by elsiosanchez**
Support to Search forms
`C_BPartner`
`C_Invoice`
`M_Product`
`A_Asset`
`C_Order`

https://user-images.githubusercontent.com/45974454/190498164-503a3de5-74db-4080-894e-8a8fbce8d5f9.mp4

image

### CODE REVIEW

File: components/ADempiere/FieldDefinition/FieldSearch/BusinessPartnerInfo/index.vue [hunk 1]
```diff
@@ -108,6 +110,9 @@ export default {
         this.remoteSearch(this.displayedValue, true)
       }
     },
+    alo(a) {
```
Comment (EdwinBetanc0urt) [inline components/ADempiere/FieldDefinition/FieldSearch/BusinessPartnerInfo/index.vue]: @elsiosanchez please remove it

File: components/ADempiere/FieldDefinition/FieldSearch/mixinFieldSearch.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /**
  * ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
  * Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A. www.erpya.com
- * Contributor(s): Edwin Betancourt EdwinBetanc0urt@outlook.com https://github.com/EdwinBetanc0urt
+ * Contributor(s): Elsio Sanchez elsiosanches@gmail.com https://github.com/elsiosanchez
```
Comment (EdwinBetanc0urt) [inline components/ADempiere/FieldDefinition/FieldSearch/mixinFieldSearch.js:4]: Don't overwrite authors

File: components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/button.vue [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <!--
  ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
  Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A. www.erpya.com
- Contributor(s): Edwin Betancourt EdwinBetanc0urt@outlook.com https://github.com/EdwinBetanc0urt
+ Contributor(s): Elsio Sanchez elsiosanches@gmail.com https://github.com/elsiosanchez
```
Comment (EdwinBetanc0urt) [inline components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/button.vue:4]: Don't overwrite authors

File: components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/index.vue [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <!--
  ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
  Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A. www.erpya.com
- Contributor(s): Edwin Betancourt EdwinBetanc0urt@outlook.com https://github.com/EdwinBetanc0urt
+ Contributor(s): Elsio Sanchez elsiosanches@gmail.com https://github.com/elsiosanchez
```
Comment (EdwinBetanc0urt) [inline components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/index.vue:4]: Don't overwrite authors

File: components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/panel.vue [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <!--
  ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
  Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A. www.erpya.com
- Contributor(s): Edwin Betancourt EdwinBetanc0urt@outlook.com https://github.com/EdwinBetanc0urt
+ Contributor(s): Elsio Sanchez elsiosanches@gmail.com https://github.com/elsiosanchez
```
Comment (EdwinBetanc0urt) [inline components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/panel.vue:4]: Don't overwrite authors

File: components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/panel.vue [hunk 6]
```diff
@@ -418,7 +403,7 @@ export default {
               this.$message({
                 type: 'warning',
                 showClose: true,
-                message: this.$t('businessPartner.notFound')
+                message: this.metadata.name + this.$t('fieldFormSearch.notFound')
```
Comment (EdwinBetanc0urt) [inline components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/panel.vue:406]: Add space between field name and text

**Review by EdwinBetanc0urt — CHANGES_REQUESTED** [review]

File: components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/panel.vue (hunk 6 above)
Comment (elsiosanchez) [reply components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/panel.vue:406]: The space between the translation has already been added

File: components/ADempiere/FieldDefinition/FieldSearch/mixinFieldSearch.js (hunk 2 above)
Comment (elsiosanchez) [reply components/ADempiere/FieldDefinition/FieldSearch/mixinFieldSearch.js:4]: I did this component I don't know why your name appears

File: components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/panel.vue (hunk 5 above)
Comment (elsiosanchez) [reply components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/panel.vue:4]: I created this component from the beginning and I made a mistake when I copied the license, I needed to change the author
https://github.com/solop-develop/frontend-default-theme/commit/b334ad281cff3a588bc0692f1818caaec26d38b9#diff-0bbe621ee65a6fa36a48fb3b3d662a88dd52dc8fadba4aa13c107a15886418a1R4

File: components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/index.vue (hunk 4 above)
Comment (elsiosanchez) [reply components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/index.vue:4]: I created this component from the beginning and I made a mistake when I copied the license, I needed to change the author
https://github.com/solop-develop/frontend-default-theme/commit/b334ad281cff3a588bc0692f1818caaec26d38b9#diff-0bbe621ee65a6fa36a48fb3b3d662a88dd52dc8fadba4aa13c107a15886418a1R4

File: components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/button.vue (hunk 3 above)
Comment (elsiosanchez) [reply components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/button.vue:4]: I created this component from the beginning and I made a mistake when I copied the license, I needed to change the author
https://github.com/solop-develop/frontend-default-theme/commit/b334ad281cff3a588bc0692f1818caaec26d38b9#diff-4d34dd521cfbe0c345e82b2cde3f7769efee341c485013bffce6349d8198b65bR4

File: components/ADempiere/FieldDefinition/FieldSearch/mixinFieldSearch.js (hunk 2 above)
Comment (elsiosanchez) [reply components/ADempiere/FieldDefinition/FieldSearch/mixinFieldSearch.js:4]: I created this component from the beginning and I made a mistake when I copied the license, I needed to change the author
https://github.com/solop-develop/frontend-default-theme/commit/b334ad281cff3a588bc0692f1818caaec26d38b9#diff-4d34dd521cfbe0c345e82b2cde3f7769efee341c485013bffce6349d8198b65bR4

File: components/ADempiere/FieldDefinition/FieldSearch/BusinessPartnerInfo/index.vue (hunk 1 above)
Comment (elsiosanchez) [reply components/ADempiere/FieldDefinition/FieldSearch/BusinessPartnerInfo/index.vue]: The function has already been removed.

**Pull request merged**